aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -1053,6 +1066,39 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
         }
     }
 
+    private fun calculateInitialHTMLSHA() {
+        try {
+            // Do not recalculate the hash if it's not the first call to `fromHTML`.
+            if (initialEditorContentParsedSHA256.isEmpty() || Arrays.equals(initialEditorContentParsedSHA256, calculateSHA256(""))) {
+                val initialHTMLParsed = toPlainHtml(false)
+                initialEditorContentParsedSHA256 = calculateSHA256(initialHTMLParsed)
+            }
+        } catch (e: Throwable) {
+            // Do nothing here. `toPlainHtml` can throw exceptions, also calculateSHA256 -> NoSuchAlgorithmException
+        }
+    }
+
+    @Throws(NoSuchAlgorithmException::class)
+    private fun calculateSHA256(s: String): ByteArray {
+        val digest = java.security.MessageDigest.getInstance("SHA-256")
```

Comment:
let's make this just `MessageDigest.getInstance("SHA-256")` and add `import java.security.MessageDigest` at the beginning of the file